Xamarin.Forms.Platform.iOS/Extensions/BindableObjectExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿#if __ANDROID__
```

Comment:
Do we have any precedent for linking between platforms like this?

Reply:
We do it for the MaterialColors in the material libraries. I would hardly call that creating *precedent* though.

I just really didn't want to copy and paste the same code two places. I think the *precedent* has been to just duplicate code into every platform like the *IsOneOf* code

But.... it's duplicated 😢

Reply:
I think we would actually move this up to the Core and just set it to EditorBrowseable(Never), since there's nothing platform specific about it.

Reply:
@samhouts moved!